Students/Lupea V. Dariana/Assignment1/src/SumOfMultiples.java

```diff
@@ -0,0 +1,15 @@
+public class SumOfMultiples {
```

Comment:
Check out [this post](http://math.stackexchange.com/questions/9259/find-the-sum-of-all-the-multiples-of-3-or-5-below-1000) and see how you can change this from `O(n)` to constant time!